.travis.yml

```diff
@@ -28,11 +28,9 @@ cache:
   directories:
     - $HOME/.ivy2/cache
     - $HOME/.sbt/boot
-    - $HOME/.jabba/jdk
```

Comment:
Where does the JDK get cached with sdkman?
Or does this even import https://github.com/scala/scala-dev/blob/scala-dev/travis/default.yml#L29 ?

Reply:
I think it should but I haven't checked - I guess perhaps we could drop this whole section here.

Reply:
Can't we remove `before_cache` and `cache` and use the values provided by `scala/scala-dev`'s default TravisCI setup?

Side note: IDK how redefinition/overloading works when using YML imports in Travis.